package.json

```diff
@@ -3,22 +3,28 @@
   "version": "1.0.0",
   "description": "This project was created to redesign the rbglive page (https://live.rbg.tum.de/). It uses HTML-Scraping to get the necessary information from the original page and then wraps it into a modern, dark-themed UI. Currently you can watch live streams and VoDs via the custom UI, Archives are not covered yet.",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "postinstall": "npm run purgeBootstrap",
+    "purgeBootstrap": "npx purgecss --css node_modules/bootstrap/dist/css/bootstrap.min.css --content \"tmpl/*\" --output node_modules/bootstrap/dist/css/bootstrap.purged.css --safelist collapsing show"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/TUM-Dev/RBGreaterAgain.git"
   },
   "author": "",
-  "license": "GNU3",
+  "license": "GPL-3.0-only",
   "bugs": {
     "url": "https://github.com/TUM-Dev/RBGreaterAgain/issues"
   },
   "homepage": "https://stream.tum.sexy",
   "dependencies": {
-    "@material-icons/font": "^1.0.6",
+    "aos": "^2.3.4",
+    "bootstrap": "^4.5.3",
```

Comment:
So why did we now suddenly include a full fledged bootstrap? 
To compare the lighthouse results - before:
![image](https://user-images.githubusercontent.com/298860/105323934-08428d00-5bcb-11eb-9996-26179dfd41fb.png)

After:
![image](https://user-images.githubusercontent.com/298860/105323979-155f7c00-5bcb-11eb-86b4-141897c18585.png)


Especially the performance took a slight hit, which usually comes with including such big frameworks. 3.4s till first contentful paint and 5.1s on the speed index vs. 1.4s without bootstrap (pre this PR). 

I'm not complaining nor saying this is super bad, just we should be aware of this slowing down the page, just to render a few links.

Reply:
If its the grid you're after, we could consider just including that and skip using the entire thing + javascript.

Reply:
Did you audit this on my server? That would be very slow, he's an old boy. Locally (after slight changes) I get:
![Screenshot_20210121_101951](https://user-images.githubusercontent.com/44805696/105330144-3f686c80-5bd2-11eb-8112-df2d207fec14.png)


What I was after is the accordion and the popup for the playback rate. This should be easy to rewrite in plain js: https://medium.com/dailyjs/mimicking-bootstraps-collapse-with-vanilla-javascript-b3bb389040e7 
I'm afraid though that I don't have the time for it just yet.

Reply:
Probably not worth the effort. Looks good to me as is.